platform/pensando/sonic-platform-modules-dpu/dpu/service/dpu-db-util.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Pensando DPU health status provider and chassis state db util service
+Requires=system-health.service
+After=system-health.service
+
+[Service]
+ExecStartPre=/bin/sleep 75
+ExecStart= python3 /usr/local/bin/dpu_db_util.py &
```

Comment:
Why is there a `&` at the end? That's Bash job control syntax, and shouldn't be present here.

This should also be an absolute path to `python3` or just `/usr/local/bin/dpu_db_util.py` (if the shebang is appropriately set).

Reply:
Removed this as well. Latest code of dpu_db_util.py doesn't require it anymore